core/src/main/scala/org/apache/spark/scheduler/TaskSetManager.scala

```diff
@@ -98,6 +84,14 @@ private[spark] class TaskSetManager(
   var totalResultSize = 0L
   var calculatedTasks = 0
 
+  val taskSetBlacklistOpt: Option[TaskSetBlacklist] = {
```

Comment:
I too go back and forth on naming `Option`s, and haven't yet hit upon a convention that is entirely satisfactory.  Another `Option` option (uh.... yeah) is `maybeFoo`, which we've also used within the Spark codebase.